parachain/pallets/ethereum-beacon-light-client/Cargo.toml

```diff
@@ -0,0 +1,57 @@
+[package]
+name = "snowbridge-ethereum-beacon-light-client"
+description = "Snowbridge Beacon Light Client Pallet"
+version = "0.0.1"
+edition = "2021"
```

Comment:
The reason why the build is still failing after adding edition 2021 is because the github workflow doing the build is using an older version of the rust toolchain. 

https://github.com/Snowfork/snowbridge/blob/main/.github/workflows/parachain.yml#L37-L39

This commit is the version I used for edition 2021 in the `v0.9.17` PR that is not yet merged in.
https://github.com/Snowfork/snowbridge/pull/591/commits/7167fdcb67e2d83b00a92944aab17b5f9eed2425

Reply:
Thanks Alistair! 👍  So the edition 2021 should work after the rust toolchain upgrade? I've started updating the other crate's editions too: https://github.com/Snowfork/snowbridge/pull/605/files